submissions/april-1st/submission-april-1st.md

```diff
@@ -0,0 +1,208 @@
+# April 1st Mod Feature
+
+(v 1.2)
+
+
+
+## Working title: Funny Factorio Fan Formulations
+
+
+
+On the special occasion of Friday April 1st, we are proud to present some humorous picks from Factorio's vast modding scene. We will start with mods that only change visuals or audio to add some unique flavors to your eningeering experiences. Afterwards, we look at some crazy new weapons that spice up combat. Finally, we feature a mod with a level of transportation pizzaz that Willy Wonka himself would be proud of.
+
+
+
+### Sights and Sounds
+
+The mod portal features quite a few mods small and large that offer changes to visuals and/or audio. Here is a fun selection:
+
+
+
+**Emotes and Speech Bubbles** 
```

Comment:
I think we can do the headings like we did in the Modding Behind The Scenes post, where we have the mod author in the header. https://alt-f4.blog/ALTF4-55/